### DISCUSSION TITLE
fine_to_coarse interpolation

### DISCUSSION THREAD

**Discussion by seanconverge** [AMReX-Codes/amrex#3314] [Q&A] [answered]
During develop my own CFD code based on AMReX, I want to transfer the data on the fine level to the coarse level with biquadratic
formula. However, I only find the amrex::average_down function is emoloyed in the example codes, and I didn't see any related functions in the src. Are there some fine_to_coarse functions can be used?

Thanks in advance.

**Comment by WeiqunZhang** [comment, marked as answer]
You can implement what you need following the example of average_down functions.

Those average_down functions are usually sufficient for many AMReX application codes because for finite-volume methods the states are cell averaged values (even for high-order say 4th-order), and for finite-difference methods we can put the states on nodes instead of cell centers.

**Reply by seanconverge** [reply]
Thanks for your reply. I will have a try following the example of average_down functions.
